tests/unit/test_routing_fixes.py

```
@@ -0,0 +1,102 @@
+
+    def test_root_endpoints_before_api_init_in_test_files(self):
+        """Test that test files have root endpoints registered before Flask-RESTX init."""
+        test_files = [
+            'deployment/cloud-run/test_swagger_debug.py',
+            'deployment/cloud-run/test_routing_debug.py',
+            'deployment/cloud-run/test_debug_server.py',
+            'deployment/cloud-run/test_routing_minimal.py'
+        ]
+
+        for test_file in test_files:
+            file_path = os.path.join(os.path.dirname(__file__), '..', '..', test_file)
```

Ruff (0.12.2)

`os.path.join()` should be replaced by `Path` with `/` operator

(PTH118)